app/models/user.rb

```diff
@@ -3,9 +3,13 @@ class User < ActiveRecord::Base
   enum role: [:attendee, :event_staff, :event_manager, :super_admin ]
 
   #associations
-  has_many :attendees, class: Attendee, foreign_key: 'user_id'
+  # has_many :attendees, class: Attendee, foreign_key: 'user_id'
   has_many :events, foreign_key: 'event_manager_id'
-  has_many :events_attending, through: :attendees, source: 'event'
+  # has_many :events_attending, through: :attendees, source: 'event'
+
+  has_many :bookings
+  has_many :user_tickets, through: :bookings
+  has_many :events_attending, through: :bookings, source: 'event'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.